glusterd2/brick/glusterfsd.go

```diff
@@ -56,7 +58,7 @@ func (b *Glusterfsd) Args() string {
 
 	brickPort := strconv.Itoa(pmap.AssignPort(0, b.brickinfo.Path))
 
-	volFileID := b.brickinfo.VolumeName + "." + gdctx.MyUUID.String() + "." + brickPathWithoutSlashes
+	volFileID := b.brickinfo.VolumeName + "." + strconv.FormatBool(b.SnapVol) + "." + gdctx.MyUUID.String() + "." + brickPathWithoutSlashes
```

Comment:
`strconv.FormatBool` returns `true` or `false` as string. That's not what you want here:

`vol1.true.uuid.brick-path`